library/src/components/Dashboard/CircleChart.tsx

```diff
@@ -0,0 +1,68 @@
+interface CircleData {
+  key?: string;
+  percentage?: string;
+  color: string;
+}
+
+export interface CircleProps {
+  equivalentSlices: boolean;
+  data: Array<CircleData>;
+}
+
+export const CircleChart = (props: CircleProps) => {
+  const { equivalentSlices, data } = props;
+  const radio = 110;
+  const circumference = Math.PI * 2 * radio;
+  if (equivalentSlices) {
+    const segmentLength = circumference / props.data.length;
+
+    return (
+      <svg width='250' height='250' style={{ display: 'flex' }}>
```

Comment:
Set width and heigth as props with default values
```suggestion
      <svg width={width} height={height} style={{ display: 'flex' }}>
```